tests/tests/bitasset_tests.cpp

```diff
@@ -553,6 +553,34 @@ class bitasset_evaluator_wrapper : public asset_update_bitasset_evaluator
    }
 };
 
+struct assets_922_931
+{
+   const asset_object* bit_usd;
+   const asset_object* bit_usdbacked;
+   const asset_object* blah;
+   const asset_object* sixbit;
+   const asset_object* prediction;
+};
+
+assets_922_931 create_assets_922_931(database_fixture* fixture)
+{
+   assets_922_931 asset_objs;
+   BOOST_TEST_MESSAGE("Create USDBIT");
+   asset_objs.bit_usd = &fixture->create_bitasset("USDBIT", GRAPHENE_COMMITTEE_ACCOUNT);
+
+   BOOST_TEST_MESSAGE("Create USDBACKEDBIT");
+   asset_objs.bit_usdbacked = &fixture->create_bitasset("USDBACKEDBIT", GRAPHENE_COMMITTEE_ACCOUNT, 100, charge_market_fee, 2, asset_objs.bit_usd->get_id());
+
+   BOOST_TEST_MESSAGE( "Create user issued BLAH" );
+   asset_objs.blah = &fixture->create_user_issued_asset( "BLAH" );
+
+   BOOST_TEST_MESSAGE( "Create a bitasset with a precision of 6" );
+   asset_objs.sixbit = &fixture->create_bitasset("SIXBIT", GRAPHENE_WITNESS_ACCOUNT, 100, charge_market_fee, 6);
+
+   BOOST_TEST_MESSAGE( "Create Prediction market coin with precision of 6, backed by SIXBIT" );
+   asset_objs.prediction = &fixture->create_prediction_market("PREDICTION", GRAPHENE_WITNESS_ACCOUNT, 100, charge_market_fee, 6, asset_objs.sixbit->get_id());
+   return asset_objs;
+}
```

Comment:
Fine so far, but there is a potential issue in the future, since all pointers to new objects will become invalid after generated a block.